api/authentication_test.go

```diff
@@ -28,7 +29,7 @@ func TestMiddleWareAuthenticationReturnsUnauthorised(t *testing.T) {
 		So(err, ShouldBeNil)
 		w := httptest.NewRecorder()
 		auth.Check(mockHTTPHandler).ServeHTTP(w, r)
-		So(w.Code, ShouldEqual, http.StatusUnauthorized)
+		So(w.Code, ShouldEqual, http.StatusNotFound)
```

Comment:
Update the convey message in this test to reflect the change

Reply:
done